centinel/as_info.py

```diff
@@ -0,0 +1,64 @@
+#
+# Abbas Razaghpanah (arazaghpanah@cs.stonybrook.edu)
+# March 2015, Stony Brook University
+#
+# as_info.py: a library to lookup AS number and AS owner information
+# for IP addresses.
+#
+
+from netaddr import IPNetwork, IPAddress
```

Comment:
Can you provide more information about this library? What reason do we have to believe that it is accurate? I have an older db from MaxMind that will let us do ASN lookups instead if we prefer. I don't care about the exact library we use, but we need to be able to make claims that our lookups are accurate.